seatunnel-api/src/main/java/org/apache/seatunnel/api/options/EnvCommonOptions.java

```diff
@@ -107,4 +107,10 @@ public class EnvCommonOptions {
                     .mapType()
                     .noDefaultValue()
                     .withDescription("Define the worker where the job runs by tag");
+
+    public static Option<String> REF_PATH =
+            Options.key("__st_config_ref_path__")
```

Comment:
`__st_config_ref_path__` looks weird, how about naming it to `config_ref_path`?

Reply:
Not surprising, he cannot conflict with normal attributes